src/bin/parser.js

```diff
@@ -401,4 +319,29 @@ export default class Harmonic {
             console.log(clc.info(`${lang}/rss.xml file successfully created`));
         });
     }
+
+    sortByDate(files) {
+        Object.values(files).forEach((filesArray) => filesArray.sort(this.sort));
+        return files;
+    }
+
+    sort(a, b) {
```

Comment:
This function doesn't seem very useful on its own.
Since we're refactoring, maybe we can turn this into an anonymous function inside `sortByDate`?